liferay-echo/liferay-portal
# LPS-176193 Welcome Site initializer should create its images in specific folder

## Motivation

[Link to ticket](https://issues.liferay.com/browse/LPS-176193)

The proposal is to move the default images inside a folder called "Provided by Liferay" to align with the OOTB and also, so that the DM root folder doesn't look too wild and untidy to customers.

## Change summary:

* Move images to the Provided by Liferay folder and adapt page definition


## Test Scenario

* Start a vanilla bundle or create a new site based on Welcome Site Initializer
* Go to Documents and Media
* A folder "Provided by Liferay" should be present with several images
* Go to Home Page, the tree image should be shown
* Edit the default 404 utility page, the images should be shown
